src/styles/_class-overrides.scss

```diff
@@ -315,3 +321,137 @@
 .graphic_image--circle {
   border-radius: 50%;
 }
+
+//mobile & tablet view
+@media (max-width: 1023px) {
```

Comment:
In a future PR I'll try to clean up the media queries using more custom mixins or functions